### ISSUE TITLE
Namibia dashboard summary table - separate % Refused and % Locked columns by first visit vs. mop-up

### ISSUE THREAD

**Issue by sbabur** [disarm-platform/user-requests-and-feedback#87]
Currently we have the following 2 columns on the far right of the dashboard:

% Refused (# structures unsprayed where Reason == Refused) and 
% Locked (# structures sprayed where Reason == Locked)

We want to split this into:

% Refused First Visit (# structures unsprayed (first visit) where Reason == Refused) 
% Refused Mop-up (# structures unsprayed (mop-up) where Reason == Refused) 
% Locked First Visit (# structures sprayed (first visit) where Reason == Locked)
% Locked Mop-up (# structures sprayed (mop-up) where Reason == Locked)

This is dependent on #86 where there will be separate questions for number unsprayed first visit vs. number unsprayed mop-up

**Comment by sbabur** [comment]
@onlyjsmith @TNgidi Only the % Refused First Visit column is getting calculated:

![image](https://user-images.githubusercontent.com/37877007/43259288-e40a5282-90d5-11e8-8f15-814b96d44557.png)

[nam_responses_2018-07-26_1326_Olwoole.xlsx](https://github.com/disarm-platform/user-requests-and-feedback/files/2231712/nam_responses_2018-07-26_1326_Olwoole.xlsx)

**Comment by sbabur** [comment]
@TNgidi @Nicolaidavies Hope this helps:

**% refused first visit:** number_unsprayed, where reasons_for_not_spraying == refused
**% refused mop-up:** number_unsprayed_mopup, where reasons_for_not_spraying == refused
**% locked first visit:** number_unsprayed, where reasons_for_not_spraying == locked_or_no_one_home
**% locked mop-up**: number_unsprayed_mopup, where reasons_for_not_spraying == locked_or_no_one_home

**Comment by sbabur** [comment]
@TNgidi Validated in production, can close
